variables.tf

```diff
@@ -161,10 +161,10 @@ variable "use_fips_ssl" {
   default     = "false"
 }
 
-variable "wlm_json_configuration" {
+variable "wlm_configuration" {
   description = "Configuration bits for WLM json. see https://docs.aws.amazon.com/redshift/latest/mgmt/workload-mgmt-config.html"
-  type        = string
-  default     = "[{\"query_concurrency\": 5}]"
+  type        = map(string)
```

Comment:
`{ query_concurrency = 5 }` is `map(number)`, `[{ query_concurrency = 5 }]` is `list(map(number))`.

Normally, if you don't know what type of value it should be, set `string` (as it is already) or `map(any)` or just `any`.

I propose to change this type to be `list(map(any))` and use `jsonencode` as you do already.

Reply:
done

Reply:
done